web/client/components/geostory/media/index.js

```diff
@@ -30,14 +33,15 @@ const LoaderComponent = () => <div className="ms-media-loader"><Loader size={52}
  * @prop {string} type one of 'image' or 'map' (used when mediaType is equal to undefined)
  * @prop {number} debounceTime debounce time for lazy loading
  */
-export const Media = ({ debounceTime, ...props }) => {
+const Media = ({ debounceTime, mediaTypesMap = typesMap, ...props }) => {
     // store all ids inside an immersive section
     // in this way every media is loaded only when in view
     const [loading, onLoad] = useState({});
     const isLoading = loading[props.id] === undefined
         ? true
         : loading[props.id];
-    const MediaType = typesMap[props.mediaType || props.type] || Image;
+    const MediaType = mediaTypesMap[props.mediaType || props.type] || Image;
```

Comment:
If you pass here the MediaViewer, now you can only use <MediaViewer type={props.mediaType}  ...>